source/cloud/azure/azureml.md

```diff
@@ -102,7 +101,9 @@ ml_client = MLClient.from_config(
 
 You will need to create a [compute target](https://learn.microsoft.com/en-us/azure/machine-learning/concept-compute-target?view=azureml-api-2#azure-machine-learning-compute-managed) using Azure ML managed compute ([AmlCompute](https://azuresdkdocs.blob.core.windows.net/$web/python/azure-ai-ml/0.1.0b4/azure.ai.ml.entities.html)) for remote training.
```

Comment:
Should we say let's create or something that makes it clear to follow the next steps, instead of "You need to..." which reads as I need to do this before following along.

Reply:
Sure sounds good